### PR TITLE
Infrastructure: Update build and test process for default branch rename to main

### LINKED ISSUE #1421: Rename the master branch
`master` → `main` or `master` → `primary`.

Resources:

- [The NoHarm project renamed their “master” branch to “publish”](https://github.com/raisely/NoHarm/issues/55)

### PR DESCRIPTION
**Pull request by jesdaigle**
Resolves #1421

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-[![Build Status](https://travis-ci.com/w3c/aria-practices.svg?branch=master)](https://travis-ci.com/w3c/aria-practices)
+[![Build Status](https://travis-ci.com/w3c/aria-practices.svg?branch=main)](https://travis-ci.com/w3c/aria-practices)
```
Comment (nschonni) [inline README.md:1]: This should probably just be deleted, since Travis isn't used anymore

File: README.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ offending character is indicated by the number `8` after the colon. Change the
 variable `value_nodes` to `valueNodes` in your source file to eliminate this
 error.
 
-To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](https://github.com/w3c/aria-practices/blob/master/.eslint.json) file in the root of the project.
+To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](https://github.com/w3c/aria-practices/blob/main/.eslint.json) file in the root of the project.
```
Comment (nschonni) [inline README.md:104]: Not sure if the relative link works with dotfiles in the Markdown auto-linking or not
```suggestion
To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](.eslint.json) file in the root of the project.
```

File: README.md (hunk 1 above)
Comment (jesdaigle) [reply README.md:1]: Updated PR incoming. :)

File: README.md (hunk 2 above)
Comment (jesdaigle) [reply README.md:104]: > Could use the relative link, but I'm not sure if the relative link works with dotfiles in the Markdown auto-linking or not

I didn't see anything in the documentation that indicates that they wouldn't. https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes

File: README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-[![Build Status](https://travis-ci.com/w3c/aria-practices.svg?branch=master)](https://travis-ci.com/w3c/aria-practices)
+
```
Comment (nschonni) [inline README.md:1]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
```
Comment (nschonni) [inline README.md:2]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -101,7 +101,7 @@ offending character is indicated by the number `8` after the colon. Change the
 variable `value_nodes` to `valueNodes` in your source file to eliminate this
 error.
 
-To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](https://github.com/w3c/aria-practices/blob/master/.eslint.json) file in the root of the project.
+To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](.eslint.json) file in the root of the project.
```
Comment (nschonni) [inline README.md:104]: Opps, just say the relative link works, but the filename was wrong before
```suggestion
To see the complete list of style rules that are applied by ESLint, review the [.eslintrc.json](.eslintrc.json) file in the root of the project.
```

**Review by nschonni — APPROVED** [review]

**Pull request merged**